.eslintrc

```diff
@@ -1,3 +1,10 @@
 {
-	"parser": "babel-eslint"
+	"extends": "humanmade",
+	"globals": {
+		"Altis": "readonly"
```

Comment:
Do we need this? I think using `window.Altis` instead avoids the need to change this?